backend/src/plugins/ModActions/types.ts

```diff
@@ -25,9 +25,9 @@ export const zModActionsConfig = z.strictObject({
   tempban_message: z.nullable(z.string()),
   default_reasons: z.nullable(
     z.object({
-      mute: z.string(),
-      kick: z.string(),
-      ban: z.string(),
```

Comment:
Instead of limiting the reason in `formatReasonWithAttachments` it might be better to do it here